src/types/interfaces.ts

```diff
@@ -58,6 +58,11 @@ export enum FilterOptionsKeywords {
   TEAM = 'ius',
 }
 
+export enum handleSelectorComponentModes {
+  CreateTaskFieldUpdate,
```

Comment:
We shouldn't be adding an enum type to an interfaces file imo, since we are grouping them based on their behavior